### DISCUSSION TITLE
Allow us to order what we want to see in our feed first.

### DISCUSSION THREAD

**Discussion by hennysmafter** [dailydotdev/daily#1220] [Feature Requests]
We can set a ton of tags of items that we are interested in, but currently it just randomly shows our feed from all these items. I don't have time to see every post, and I am just interested in many things. 

It would be great if we could order the results by tags. So I suggest putting a tab in the My Feed Settings that says ordering. 
In this tab it loads all the selected tags and the user can simply grab them and order them in the way they want. 

Now, when the feed is loaded and there are new items to show for a given tag, it looks at the ordering.

For instance, for myself, I would put web design / development tags before tech news. So if there is anything new in the tags web design I want to see those items before I see the new items in tech news.

This way I can very quickly see what I am most interested in and then if I have more time see the rest.

**Comment by idoshamun** [comment]
Hi @hennysmafter,

We recently released custom feeds which should help with this issue.
You can create multiple feeds with different tags allowing to browse only the tags you want when you want.
Would that help?
